modules/stanford_courses_importer/config/install/stanford_courses_importer.importer_settings.yml

```diff
@@ -0,0 +1,3 @@
+urls:
+  - 'https://explorecourses.stanford.edu/search?view=xml-20200810&academicYear=&page=0&q=ACCT&filter-departmentcode-ACCT=on&filter-coursestatus-Active=on'
```

Comment:
I'm guessing this was something left over from the H&S stuff? We shouldn't need this.

Reply:
Ah, yeah, I had it in there for testing.  thanks